source/docs/software/vision-processing/index.rst

```diff
@@ -5,5 +5,4 @@ Vision Processing
    :maxdepth: 2
 
    introduction/index
```

Comment:
I think we need something in the vision section that mentions GRIP. Can we add a ¶ to the intro?